ql/src/test/results/clientpositive/druid/druidmini_joins.q.out

```diff
@@ -223,8 +228,8 @@ GROUP BY `tbl1`.`username`
 POSTHOOK: type: QUERY
```

Comment:
can we add an order by here to avoid test flakyness

Reply:
added